RXPatchLib/WebPatchSource.cs

```diff
@@ -193,14 +192,6 @@ private void UnlockDownload()
                                     $"Error while attempting to transfer the file.\r\n{e.Message}\r\n{e.StackTrace}");
                                 cancellationToken.ThrowIfCancellationRequested();
                                 AXDebug.AxDebuggerHandler.Instance.RemoveDownload(guid);
-

```

Comment:
If we're receiving 400/client errors from the server, they're not going to be resolvable and we should continue fail fast here.